app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -987,6 +988,7 @@ $include-html-global-classes: $include-html-classes;
 
 // We use these to control the background and border styles
 // $panel-bg: scale-color($white, $lightness: -5%);
+$panel-bg: scale-color(#a6ad9f, $lightness: -5%);
```

Comment:
Color literals like `#a6ad9f` should only be used in variable declarations; they should be referred to via variable everywhere else.